### PR TITLE
try to initialize WS Client 5 times

### PR DESCRIPTION
**Pull request by TomasTomecek**
and workaround the 'No route to host' errors in prod

### CODE REVIEW

File: sandcastle/api.py [hunk 1]
```diff
@@ -427,18 +427,27 @@ def run(self, command: Optional[List] = None):
...
-            self.k8s_namespace_name,
-            command=command,
-            stdin=False,
-            stderr=True,
-            stdout=True,
-            tty=False,
-            _preload_content=preload_content,  # <<< we need a client object
-            _request_timeout=WEBSOCKET_CALL_TIMEOUT,
-        )
+        for i in range(5):
```
Comment (lachmanfrantisek) [inline sandcastle/api.py]: Maybe some constant?

Comment (TomasTomecek) [reply sandcastle/api.py]: uuhhhhhhh...

Comment (lachmanfrantisek) [reply sandcastle/api.py]: sorry, just a suggestion

File: sandcastle/api.py [hunk 2]
```diff
@@ -427,18 +427,27 @@ def run(self, command: Optional[List] = None):
...
+                    stdout=True,
+                    tty=False,
+                    _preload_content=preload_content,  # <<< we need a client object
+                    _request_timeout=WEBSOCKET_CALL_TIMEOUT,
+                )
+            except ApiException as ex:
+                # in packit-service prod, occasionally 'No route to host' happens here
+                # let's try to repeat the request
+                logger.error("exception while initiating WS Client: %r", ex)
+                time.sleep(i + 1)
```
[the remark this answers is not in the document]
Comment (TomasTomecek) [reply sandcastle/api.py]: no!

File: sandcastle/api.py [hunk 3]
```diff
@@ -74,6 +74,8 @@
 logger = logging.getLogger(__name__)
 
 WEBSOCKET_CALL_TIMEOUT = 30.0  # seconds
+# try to initiate WS Client 5 times
+MAGIC_KONSTANT = 5
```
Comment (lachmanfrantisek) [inline sandcastle/api.py]: If you had chosen a better name, you wouldn't need the comment...;-)

Comment (TomasTomecek) [reply sandcastle/api.py]: there is no better name for that constant :P

**Pull request merged**